src/screens/Login/index.tsx

```diff
@@ -26,12 +28,10 @@ function Login() {
   } = useForm<User>();
 
   const { mutate, isLoading, isError } = useMutation((values: User) => signIn(values), {
-    onSuccess: ({ headers }) =>
-      console.log({
-        accessToken: headers?.['access-token'],
-        uid: headers?.uid,
-        client: headers?.client
-      }),
+    onSuccess: ({ headers }) => {
+      LocalStorageService.setValue('access-token', headers?.['access-token']);
+      history.push(PATHS.home);
```

Comment:
This works fine but you will need to get the headers and use them every time an api request is needed, so it may turn cumbersome.
The acquired headers can be added to the api (and removed from). Check api sauce's `api.setHeader`